### ISSUE TITLE
Spectral Clustering vs. K-means (Pavlovic)

### ISSUE THREAD

**Issue by dlee138** [openconnectome/Statistical-Connectomics-Class#167]
What was the strategic reason as to why spectral clustering was used in the Pavlovic paper?  Can someone give me a brief overview of how spectral clustering works and how it differs from some of the other methods we discussed in class such as k-means?

**Comment by ajulian3** [comment]
K Means deals with compactness of spatial data while spectral clustering deals with connectivity. Spectral Clustering is important when determining neural network connectivity.

**Comment by dlee138** [comment]
Yes, so K-means focuses more on "closeness" while spectral deals with connectivity.

**Comment by michaelseung** [comment]
I think spectral clustering is also more useful when dealing with higher dimensions with the use of eigenvalues to cluster in fewer dimensions.
